cmd/juju/controller/register.go

```diff
@@ -323,7 +323,13 @@ func (c *registerCommand) updateController(
 	}
 	for name, ctl := range all {
 		if ctl.ControllerUUID == controllerDetails.ControllerUUID {
-			// TODO(rogpeppe) lp#1614010 Succeed but override the account details in this case?
+			ctx.Warningf("This controller has already been registered on this client as %q.", name)
```

Comment:
I find the "WARNING" on each line makes it difficult to read. How about just a WARNING on the first line, followed by "This controller has already...", and then the remainder of the message on proceeding lines without any prefix. Like:

```
WARNING This controller has already been registered on this client as <foo>.

To login user <bar>, run ...
To update controller details and login as user <bar>:
...
```